### ISSUE TITLE
MSI automated installers

### ISSUE THREAD

**Issue by mcandre** [adoxa/ansicon#56]
`ansicon.exe -i` is cool, but I thought I would automate installation a bit more. I created some MSI installers that automatically place the associated ansicon files in `Program Files`, add `ansicon.exe` to `PATH`, and call `ansicon.exe -i` as a post install script.

https://github.com/mcandre/ansicon-win

**Comment by Rask** [comment]
Is there an uninstaller to go with it?

**Comment by mcandre** [comment]
Yes. All MSIs include uninstallation via Windows Programs and Features.

**Comment by mcandre** [comment]
Yes. All MSI installers implicitly include an uninstaller, accessible via Windows Programs and Features.
